envi/__init__.py

```diff
@@ -262,16 +265,15 @@ def getOperValue(self, op, emu=None):
         NOTE: This API may be passed a None emu and should return what it can
               (or None if it can't be resolved)
         """
-        print("%s needs to implement getOperValue!" % self.__class__.__name__)
-        return None
+        raise NotImplementedError("%s needs to implement getOperValue!" % self.__class__.__name__)
```

Comment:
So it's a lot stricter about forcing Operands to implement getOperValue. I don't particularly have any good tests that force this condition, and the unittests don't seem to complain (despite our lower coverage).

I'm still unsure about it though, so I wanted to pose the question of whether or not this is okay to do.

Reply:
seems reasonable to expect Operands to implement getOperValue(), even if they override it to return None because it is meaningless.

getOperAddr, OTOH, would not make sense to do.

Reply:
:+1: